code/modules/projectiles/projectile_parent.dm

```diff
@@ -933,14 +951,19 @@ datum/projectile/snowball
 	P.yo = yo
 	return P
 
-/proc/stun_bullet_hit(var/obj/projectile/P, var/mob/living/M)
-	if (ishuman(M) && !isdead(M))
-		var/mob/living/carbon/human/H = M
-		H.changeStatus("stunned", max(H.getStatusDuration("stunned"), P.power*2.5))
-	else if (isrobot(M))
-		var/mob/living/silicon/robot/R = M
-		R.changeStatus("stunned", max(R.getStatusDuration("stunned"), 50))
-	M.force_laydown_standup()
+/proc/stun_bullet_hit(var/obj/projectile/O, var/mob/living/L)
+#ifdef USE_STAMINA_DISORIENT
+	L.do_disorient(clamp(O.power*4, O.proj_data.power*2, O.power+80), weakened = O.power*2, stunned = O.power*2, disorient = min(O.power, 80), remove_stamina_below_zero = 0)
+	L.emote("twitch_v")// for the above, flooring stam based off the power of the datum is intentional
+#else
```

Comment:
no need to use  the `#ifdef USE_STAMINA_DISORIENT` anymore, just delete the junk within the `#else` and keep the do_disorient() stuff

Reply:
done